draft-ietf-webpush-protocol.xml

```diff
@@ -529,11 +560,15 @@ TTL = 1*DIGIT
         contents of push messages as they are sent by application servers.
       </t>
       <t>
-        Each push message is pushed in response to a synthesized GET request.
-        The GET request is made to the push message resource that was created by
-        the push service when the application server requested message delivery.
-        The response body is the entity body from the most recent request sent
-        to the push resource.
+        Each push message is pushed as the response to a synthesized GET request
+        in a PUSH_PROMISE.  This GET request is made to the push message resource
+        that was created by the push service when the application server requested
+        message delivery.  The response headers SHOULD provide a URI for the push
+        resource corresponding to the push message subscription in a link relation
+        of type "urn:ietf:params:push" and a URI for the receipt subscribe resource
+        in a link relation of type "urn:ietf:params:push:receipt". The response body

```

Comment:
I noted the same thing.  I think that Brian is copying what we already recommended.  We can probably move the receipt link relation to the subscription [set] resource.  I'll open an issue.